### DISCUSSION TITLE
Can't use <views:TodoListView/> in Window on the Todo List tutorial?

### DISCUSSION THREAD

**Discussion by red-moss2** [AvaloniaUI/Avalonia#7970] [Q&A]
I'm at this step of the Todo List tutorial: https://docs.avaloniaui.net/tutorials/todo-list-app/wiring-up-the-views

It says that I can now insert the Todo List by setting the attribute `Content="{Binding List}"` on the Window. But this doesn't seem too nice as I would prefer to insert the view inside so I have better control over it.

But if I use `<views:TodoListView/>` within the Window, the items don't appear (the button does).

I can get around this by adding `<UserControl Content="{Binding List}"></UserControl>`, but I was curious as to why the list items from the Model don't appear?

**Comment by maxkatz6** [comment]
Can you show what was your code that doesn't work? In general, to display items you need to use ItemsControl (or any other ItemsPresenter-based control, like ListBox). And to not forget about DataContext that should pass to the TodoListView.
